docs/REST API/rest-api.md

```diff
@@ -8767,6 +8767,9 @@ Uploads a script, making it available to run on hosts assigned to the specified
 | script          | file    | form | **Required**. The file containing the script.    |
 | team_id         | integer | form | _Available in Fleet Premium_. The team ID. If specified, the script will only be available to hosts assigned to this team. If not specified, the script will only be available to hosts on **no team**.  |
 
+Script line endings are automatically converted from CRLF to LF for compatibility with both
```

Comment:
I'd consider including a link for the user-facing docs
```suggestion
Script line endings are automatically converted from [CRLF to LF](https://en.wikipedia.org/wiki/Newline) for compatibility with both
```

Reply:
Seems reasonable. Was looking at that entry to see if there was a specific section that would've been better to link directly to, but the answer was "not really."